common/gnome-shell/3.18/sass/_common.scss

```diff
@@ -687,8 +687,8 @@ StScrollBar {
   -arrow-background-color: $osd_bg_color;
   -arrow-border-width: 1px;
   -arrow-border-color: transparentize(black, 0.6);
-  -arrow-base: 5;
-  -arrow-rise: 5;
+  -arrow-base: 20px;
+  -arrow-rise: 10px;
 }
```

Comment:
With this, and https://github.com/jnsh/arc-theme/pull/3/commits/cf0f8eb5ca74ad3277ccd231ec85c4730343de64, I'm wondering what the original design looked like on gnome-shell 3.18, when the `-arrow-base` and `-arrow-rise` definitions were added. The popup menu arrows aren't used in other gnome-shell's popup-menus, and I'm wondering if that was the intention here as well.

I don't have GNOME 3.18 set up on a VM for testing so I can't check right now. Just wondering if you investigated this further. Also what did you use as a basis for the new pixel values?

Reply:
The `-arrow-base` and `arrow-rise` properties are present in horst's theme as far back as the initial commits for version 3.14, so it's hard to see what the design change was.

Since assigning a non-zero numerical value without units to these properties is an error/typo, I was trying to figure out what the original design intent was. The `-arrow-base` and `arrow-rise` properties appear 4 times in the Arc gnome shell theme:
- `.popup-menu-boxpointer`: the properties are zeroed out, so obviously the intent was to hide the arrow
- `.keyboard-subkeys`: explicit values assigned (20px base, 10px rise) to ensure arrow is displayed
- `.candidate-popup-boxpointer` and `.app-folder-popup`: assigning a value of 5 to the properties is an error, so currently the Arc theme falls back on the values in the default shell theme (in both cases `-arrow-base: 24px` and `-arrow-rise: 11px`) and the arrow ends up being displayed in both cases. Since horst then modified some of the other `-arrow-*` properties in `.candidate-popup-boxpointer` and `.app-folder-popup`, I went with the assumption that the intent was to have the arrow displayed in both cases. The pixel values I used (20px/10px) are consistent with those used by the Arc theme in `.keyboard-subkeys`, for uniformity.

Reply:
Thanks for the clarification: I'm happy to hear you considered this thoroughly :)

I think it's safe to assume that the original intention was to have the arrow-pointers visible, and this would definitely make sense at least for the `.app-folder-popup`, since without the arrow there's no way to determine which app-folder is opened if there are multiple folders side-by-side. Re-using the values from `.keyboard-subkeys` also makes sense.